### PR TITLE
use jupytext to pair notebooks

### PR DESCRIPTION
**Pull request by joamatab**

### CODE REVIEW

File: docs/notebooks/01_references.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ You can start by making a blank `KFactory` and add a single polygon to it.
 ```python
 import kfactory as kf
 
+kf.config.display_type = "image"
+
```
Comment (sebastian-goeldi) [inline docs/notebooks/01_references.md:39]: Why change to image here? The docs will already run with this behavior. If you want to reproduces locally use `export KFACTORY_DISPLAY_TYPE="image"`

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -146,6 +146,11 @@ repos:
   #   rev: 2b2f0c74acdb3de316e23ceb7dd0d7945c354050
   #   hooks:
   #     - id: reorder-python-imports
+  - repo: https://github.com/mwouts/jupytext
+    rev: 79351dfac9fde7c72cc73d553f662b4887bc63f5  # CURRENT_TAG/COMMIT_HASH
+    hooks:
+    - id: jupytext
+      args: [--from, py, --to, md]
```
Comment (sebastian-goeldi) [inline .pre-commit-config.yaml:153]: What's the purpose of this?  I don't want duplicates of py -> md in the repo. If it is simply about checking whether it is possible, the runner can take care of this already, no?

File: docs/star.md
Comment (sebastian-goeldi) [inline docs/star.md:1]: Why add the `.md` files? They were python files intentionally.

**Review by sebastian-goeldi — CHANGES_REQUESTED** [review]

**Comment by joamatab** [comment]
I see, the mkdocs was not working for me before adding the jupytext,

you are right, having `py` and `md` files does not make too much sense and is confusing

**Pull request closed without merging**